_scripts/check.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh -e
+if [[ $(which node) && $(node --version) ]]; then
+  node _scripts/check_node_version.js
+else
+  echo "install node to continue installation"
+  echo "https://nodejs.org"
+  exit 1
+fi
+
+if [[ $(which docker) && $(docker --version) ]]; then
+  docker=y
+else
+  docker=n
+fi
+
+os="$(uname -a | cut -f 1 -d ' ')"
+if [ "$os" = "Darwin" ]; then
+    if [ "$docker" = "n" ]; then
+      echo "install docker to continue installation"
+      echo "https://docs.docker.com/docker-for-mac/install/"
+      exit 1
+    fi
+
+    if [[ $(which brew) && $(brew --version) ]]; then
+      if [[ $(locate gmp) ]]; then
```

Comment:
this fails for me 

![image](https://user-images.githubusercontent.com/128755/42225355-5e3d7130-7eaa-11e8-85fc-74675c135e9d.png)

Do we need to find some other method ? Try this instead https://stackoverflow.com/a/20802425